pkg/pod/entrypoint_test.go

```diff
@@ -104,6 +105,58 @@ func TestOrderContainers(t *testing.T) {
 	}
 }
 
+func TestOrderContainersStepActions(t *testing.T) {
```

Comment:
which lines of code is this testing agains?

Reply:
good catch. I was testing the `StepName` argument but after Billy's comment, I removed it so this test was not needed. Removed.